ui/package.json

```diff
@@ -45,6 +45,7 @@
         "webpack-license-plugin": "^4.2.2"
     },
     "dependencies": {
+        "@fontsource/open-sans": "^4.5.13",
```

Comment:
Open Sansin lisäämiseen mukaan on monta tapaa. Yksi vaihtoehto olisi ollut ottaa ne mukaan lokaalisti fonttitiedostoina, mutta kun ne kerran saa npm-pakettinakin, niin why not. Optimointivaraa tässä saattaa tosin olla, sillä nyt mukaan saattaa tulla myös sellaisia versioita Open Sansista, joita ei oikeasti tarvita (kaikki eri vahvuudet, italic jne. on kukin omassa tiedostossaan. Oikeasti tarvitaan vaan tavallinen Open Sans, ja siitä voimakkuudet 400 ja 600)